cuda_core/tests/conftest.py

```diff
@@ -65,3 +65,9 @@ def clean_up_cffi_files():
             os.remove(f)
         except FileNotFoundError:
             pass  # noqa: SIM105
+
+
+def can_load_generated_ptx():
+    _, driver_ver = driver.cuDriverGetVersion()
+    _, nvrtc_major, nvrtc_minor = nvrtc.nvrtcVersion()
+    return not nvrtc_major * 1000 + nvrtc_minor * 10 > driver_ver
```

Comment:
I am a bit paranoid about operator precedence, let's make it clear
```suggestion
    return not (nvrtc_major * 1000 + nvrtc_minor * 10 > driver_ver)
```

Reply:
```
return nvrtc_major * 1000 + nvrtc_minor * 10 <= driver_ver
```
?